### DISCUSSION TITLE
keyball39 トラックボールが反応しない

### DISCUSSION THREAD

**Discussion by yoshun-jda** [Yowkees/keyball#310] [Q&A]
約1ヶ月トラックボールは反応していたのですが、突然反応しなくなりました。
OLEDの表示は0のままです。
#276 を参考に導通チェックを行いましたが、導通はしている模様です。
はんだ付けのし直しや、proMicroの交換も行いましたがダメでした。
そのほかにどのような要因が考えられるでしょうか？
よろしくお願いします。

![C587409C-0E18-43E4-A14B-2D610FE78600_1_102_o](https://github.com/Yowkees/keyball/assets/76540406/49be1e00-d00d-458a-8236-c69cb7d7c143)
![31AA5E6D-E2CC-41AD-B64C-FD7387CC45A3_1_102_o](https://github.com/Yowkees/keyball/assets/76540406/f4a4edfe-1412-46d1-a44e-e7c368654f6a)
![E42D5F50-9287-4F1D-8868-DA482D5CDE79_1_102_o](https://github.com/Yowkees/keyball/assets/76540406/20e7c87e-480a-4dba-ad6a-ea7a335e34f0)

![E5AFDB9F-EE1F-4BBE-A877-4AA87F56E29C_1_102_o](https://github.com/Yowkees/keyball/assets/76540406/d58a763b-f2d8-4e47-b126-6ba8bf935305)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

一部、はんだの量が少ないように見える箇所がありますので、それらにははんだを足した方が良いかもしれません。
導通確認はOKで、半田付けの見直しやProMicroの交換も試していただけたということで、
L字7Pコンスルーを疑ってみたいと思います。

最近あった事例で、L字コンスルーのピンが１つ変形して導通不良となっていたものがありました。横からみて曲がっているピンがないかご確認ください。また、センサー基板の挿さりが甘くても動作不良となります。

これらに着目して一度、ご確認をお願いいたします。

**Comment by yoshun-jda** [comment]
ご回答ありがとうございます。

L字コンスルーのピンが変形しているということはありませんでしたが、12Pピンソケットが少し浮いていたようで、はんだ付けし直したところ直りました。
ご迷惑おかけしました。

**Comment by Yowkees** [comment]
12Pピンソケット側に問題がありましたか。ご自身で解決いただきありがとうございました。
ぜひ末永くご愛用いただければ幸いです。
